frontend/src/new-ui/views/CategoryManagementTab.css

```
@@ -189,97 +190,387 @@
+.add-rule-btn:disabled {
+  background: #a0aec0;
+  cursor: not-allowed;
 }
 
-.test-results {
-  background: #e8f5e8;
-  border: 1px solid #c3e6c3;
+/* Rules List */
+.rules-list {
+  margin-top: 16px;
+  space-y: 12px;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)